### PR TITLE
Docs improvement

### PR DESCRIPTION
**Pull request by fabianorosas**

### CODE REVIEW

**Comment by fabianorosas** [comment]
I still need to remove in the builds repo the code which updated the packages table and the code in the infrastructure repo which called it during the daily builds.

File: README.md [hunk 1]
```diff
@@ -14,25 +18,25 @@ https://open-power-host-os.github.io/latest
...
 
-## Packages groups
+### Packages groups
 
-For convenience, a set of related packages is grouped into a
-"metapackage". The list of metapackages and the packages that are
-installed by them is the following:
+A set of related packages is grouped into a "metapackage". The list of
+metapackages and the packages that are installed by them is the
+following:
```
Comment (olavph) [inline README.md]: This should be in the previous patch.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
-# Version data and metadata for host-os projects #
+# OpenPOWER Host OS packages metadata #
```
Comment (olavph) [inline README.md]: The trailing `#` could be removed.

**Comment by fabianorosas** [comment]
> I still need to remove in the builds repo the code which updated the packages table and the code in the infrastructure repo which called it during the daily builds.

here: https://github.com/open-power-host-os/builds/pull/260

File: README.md (hunk 1 above)
Comment (fabianorosas) [inline README.md]: I just removed the "For convenince," text.

**Pull request merged**